google/cloud/bigtable/testing/table_integration_test.cc

```diff
@@ -69,12 +67,16 @@ void TableTestEnvironment::SetUp() {
   std::string const family2 = "family2";
   std::string const family3 = "family3";
   std::string const family4 = "family4";
-  bigtable::TableConfig table_config =
-      bigtable::TableConfig({{family1, bigtable::GcRule::MaxNumVersions(10)},
-                             {family2, bigtable::GcRule::MaxNumVersions(10)},
-                             {family3, bigtable::GcRule::MaxNumVersions(10)},
-                             {family4, bigtable::GcRule::MaxNumVersions(10)}},
-                            {});
+  auto constexpr kTestMaxVersions = 10;
```

Comment:
This is fine to leave, as-is. But if this change was made because of the readability-magic-numbers check, we should disable it. I think that check goes way too far and more often than not makes code less clear. Of course, we can still disable it later even with this code, so no change necessary.

Reply:
There was some repetition too. In any case, I think the warning is pretty annoying in tests, but useful in the code itself.  Others can remove it later.